EIPS/eip-5050.md

```diff
@@ -0,0 +1,321 @@
+---
+eip: 5050
+title: NFT Interaction Standard
```

Comment:
We try to discourage "standard" in titles and descriptions, since it ends up in nearly every ERC if we don't.